### PR TITLE
Update -C FAQs

### PR DESCRIPTION
**Pull request by jricher**
Adds FAQ on reaching FAL3

Adds FAQ on the authorization component (oauth access token)

### CODE REVIEW

File: faq-v1.md
Comment (jimfenton) [inline faq-v1.md]: A bit of a nit, but I'd probably refer to it as a key reference rather than a certificate reference (since it's holder-of-key, not certificate). Also would call it a fingerprint, not thumbprint. Might even say "fingerprint (hash of public key)".

**Review by jimfenton — CHANGES_REQUESTED** [review]
Minor changes suggested; let me know if you disagree.

File: faq-v1.md
Comment (jricher) [reply faq-v1.md]: The specific example here is certificate, but I can reword that to make it clear we're referencing the key from the certificate. And "thumbprint" seems to be the industry term, at least in the OAuth/OIDC world. If there's a reference for the term we can use that.

Comment (jimfenton) [reply faq-v1.md]: OK, "thumbprint" must be an OAuth thing: "key fingerprint" has 557,000 hits on Google and "key thumbprint" has 1,850. And Wikipedia defines it (on the thumbprint disambiguation page) as "a term for public key fingerprint used in Microsoft software," not that they're authoritative or anything. But if that's the correct term to use here, use it, but perhaps it should be defined.

Comment (jricher) [reply faq-v1.md]: I'm fine with "key fingerprint", and I'll change that.

Interestingly, I'm thinking Microsoft must have pushed it into the OAuth/OIDC space -- not surprising, to be honest.

Comment (jricher) [reply faq-v1.md]: Updated in current draft.

**Review by jimfenton — COMMENTED** [review]
Looks good now; like the new changes. Waiting for @DTemo to review before merging.

**Review by jimfenton — APPROVED** [review]

**Comment by jricher** [comment]
Pull request has been updated to comply with new site structure and format. Still awaiting content review and approval from @DTemo before merge.

**Comment by DTemo** [comment]
63C response changes look  good -- good to go. Sorry for the delay

**Pull request merged**
